web-docs/walkthrough/index.adoc

```diff
@@ -0,0 +1,369 @@
+In the “Search Jobs” field at the top, enter part of your username or image name to locate your image. 
+
+[NOTE]
+====
+You cannot search by the title of your process, just the username or image name.
+====
+
+image::create-job-type-search.png[]
+
+
+=== Create Input and Output Workspaces
+
+Before you can add your process as a new job type in Scale, you need to create your input and output workspaces. 
+A workspace is an external host folder or S3 bucket that can be connected to Scale. 
+A workspace may be the source location of your incoming data or the destination location for products created by your recipes, but never both. 
+Input and output must each have their own workspace.
+
+. Go to the root directory in your common data storage location and create a new folder with two subfolders. One of the subfolders will be for your input and one of the subfolders will be for output; name them accordingly.
+. Copy the input file you created in Step 1 into the input folder.
+. Go back to Scale, navigate to “System -> Workspaces” and click on the “+” to create a new workspace. Enter a title for your input workspace, and a description if you like. Under “Type,” select “Host” and enter the path to your input folder in “Host Path” to the right.
+image:create-input-workspace.png[]
+. Click “Validate” and then “Save.”
+. Repeat this process using your output folder to create an output workspace.
+
+image::create-output-workspace.png[]
+
+=== Create a Job Type
+
+A job type represents a Seed image that has been imported into Scale. To create your job type:
+
+. Navigate to “Configuration -> Job Types” then click on the “+” to create a new job type.
+. Enter the first few letters of your Docker username or image name in the “Search Jobs” field, until you locate your job. Click on that job and click “Import” in the lower right.
+. Click on the “2” (Configuration) at the left. In the “Default” box, select the output workspace you created in step 7. In the “Priority” text box, enter “201” (a normal priority). Do not select anything for “OUTPUT_FILE.”
+image:job-type-configuration.png[]
+. Click on the “3” (General Information) and pick an appropriate-looking icon for your job. The name of the icon doesn’t matter.
+image:job-type-general-info.png[]
+. Click on the “4” (Validate and Create), click “Validate” and then “Save”. This will create your job type.
+
+=== Create a Recipe to Run an Instance of a Job Type
+
+A recipe is a collection of one or more job types that interact with each other in a workflow.
+
+. Navigate to “Configuration -> Recipe Types” then click on the “+” to create a new recipe.
+. Enter a title for your recipe, and a description if you like.
+. Click on “File Inputs” in the middle right and enter a name (commonly “input_file”) to represent your input file, set “Required” to “True,” then click “+ Add File to Input” and close the window. 
```

Comment:
Due to case mismatch and challenges we've had when not set to all caps. I'd just recommend we always prefer caps.

Reply:
Good call, pushed an update to that uses `INPUT_FILE` as the example.